mobile/src/reducers/organization/organization.reducer.ts

```diff
@@ -16,7 +16,6 @@ export const organizationEpics = combineEpics(
     loadEmployeeEpic$ as any,
     loadDepartmentsEpic$ as any,
     loadChiefsEpic$ as any,
-    loadDepartmentsFinishedEpic$ as any,
     loadEmployeesForDepartmentEpic$ as any
```

Comment:
this is duplicated in people reducer... why does people reducer exist, let's put it in one folder/file for now.

Reply:
ok, is it ok to have it's code placed at employees.reducer.ts ?

Reply:
but you don't have a separate reducer, do you? You just some new epics. Reducers are side-effect-free functions, epics are side-effects.
Move your new epics either in organization.epics or in new file called employees.epics. In that case, however, you'll need to move everything that load employees in that file.